W-17935871-visualizer-tab-monitoring-duke (#50)
mulesoft/docs-reuse: modules/ROOT/partials/billing/pricing.adoc

|===

[[visualizer-widget]]

> NOTE: Anypoint Visualizer app network integration refers to an Anypoint Visualizer widget within Anypoint Monitoring. Anypoint Visualizer as a product outside of Anypoint Monitoring is available to organizations regardless of pricing plan. 

Author: ```suggestion
NOTE: Anypoint Visualizer app network integration refers to features in the *Application Network* tab within Anypoint Monitoring. Anypoint Visualizer as a product outside of Anypoint Monitoring is available to organizations regardless of pricing plan. 
```
